tests/Mutant/MutantCreatorTest.php

```diff
@@ -18,6 +18,28 @@
 
 class MutantCreatorTest extends MockeryTestCase
 {
+    private $fileLocation;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->fileLocation = __DIR__ . '/../Fixtures/MutantCreator/HasMutation/mutant.hash.infection.php';
```

Comment:
If it does create a file, this better be a temporary directory. [Should be trivial.](https://github.com/infection/infection/blob/79c873663e088a3cbaf3a7a1cd7600f25845fb90/tests/TestFramework/PhpSpec/Config/Builder/InitialConfigBuilderTest.php#L36)